ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -276,6 +276,13 @@ def NthElement(*, input, n, reverse=False, name="NthElement"):
     return ivy.astype(ivy.sort(input, descending=reverse)[..., n], input.dtype)
 
 
+@to_ivy_arrays_and_back
+def OneHot(*, indices, depth, on_value, off_value, axis=-1, name="OneHot"):
```

Comment:
We should flag that the function does only support few data types, `int32, int64, uint8`